src/v2/components/WorkspaceMembers/index.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { ChevronLeftIcon, ChevronRightIcon, LinkIcon } from '@chakra-ui/icons'
+import {
+	Box,
+	Button,
+	Center,
+	Checkbox,
+	Flex,
+	Grid,
+	GridItem,
+	Spacer,
+	Table,
+	Tag,
+	Tbody,
+	Td,
+	Th,
+	Thead,
+	Tr,
+	Wrap,
+	WrapItem,
+} from '@chakra-ui/react'
+import { capitalizeFirstLetter } from 'src/utils/generics'
+import { ApiClientsContext } from '../../../../pages/_app'
+import Loader from '../../../components/ui/loader'
+import { defaultChainId } from '../../../constants/chains'
+import { useGetWorkspaceMembersByWorkspaceIdQuery, WorkspaceMemberAccessLevel } from '../../../generated/graphql'
+import { getFormattedDateFromUnixTimestampWithYear } from '../../../utils/formattingUtils'
+import { getSupportedChainIdFromWorkspace } from '../../../utils/validationUtils'
+import InviteModal from '../InviteModal'
+
+const PAGE_SIZE = 7
+
+const userTypes = [
+	{
+		'name': 'All',
+		'accessLevels': Object.values(WorkspaceMemberAccessLevel),
+	},
+	{
+		'name': 'Administrators',
+		'accessLevels': [WorkspaceMemberAccessLevel['Admin']],
+	},
+	{
+		'name': 'Reviewers',
+		'accessLevels': [WorkspaceMemberAccessLevel['Reviewer']],
+	},
+]
+
+
+const tableHeaders = ['', 'Member', 'Role', 'Joined on']
+
+function WorkspaceMembers() {
+	const { workspace, subgraphClients } = useContext(ApiClientsContext)!
+
+	const [selectedUserTypeIdx, setSelectedUserTypeIdx] = useState(0)
+
+	const [members, setMembers] = useState<Array<any> | undefined>(undefined)
+
+	const [page, setPage] = useState(0)
+	const [hasMoreData, setHasMoreData] = useState(true)
+
+	const { data } = useGetWorkspaceMembersByWorkspaceIdQuery({
+		client:
+    subgraphClients[
+    	getSupportedChainIdFromWorkspace(workspace) || defaultChainId
+    ].client,
+		variables: {
+			workspaceId: workspace!.id,
+			first: PAGE_SIZE,
+			skip: page * PAGE_SIZE,
+			accessLevelsIn: userTypes[selectedUserTypeIdx].accessLevels,
+		},
+	})
+
+	useEffect(() => {
+		if(!data) {
+			return
+		}
+
+		setHasMoreData(data!.workspaceMembers.length >= PAGE_SIZE)
+		setMembers(data!.workspaceMembers)
+	}, [data, page, selectedUserTypeIdx])
+
+	const [isInviteModalOpen, setIsInviteModalOpen] = React.useState(false)
+
+	return (
+		<>
+			<Flex
+				direction={'column'}
+			>
+				<Flex
+					direction={'row'}
+				>
+					<Wrap>
+						{
+							userTypes.map((userType, index) => (
+								<WrapItem key={index}>
```

Comment:
I think we can use the standard tabs component instead?

https://chakra-ui.com/docs/components/tabs/usage

Will be cleaner